src/app/shared/dashboard-list-item/dashboard-list-item.component.html

```diff
@@ -33,7 +33,7 @@
         <div class="list__item__column__title col-md-12 col-6 d-lg-none">
           {{ isBorrowed ? 'REPAID' : 'INTEREST' }}
         </div>
-        <div class="col-md-12 col-6">
+        <div id="label-repaid" class="col-md-12 col-6">
```

Comment:
@martinbobbio use classname instead of IDs

Reply:
Done